File: python/tests/slugify_test.py
import challenges.slugify


def test_slugify():
    assert challenges.slugify.slugify("My Test Title") == "my-test-title"
    assert challenges.slugify.slugify("String w/ some extra &stuff'") == "string-w--some-extra--stuff"


def test_deslugify():
    assert challenges.slugify.deslugify("my-test-title") == "My Test Title"

File: elixir/apps/challenges/test/challenges/slugify_test.exs
defmodule Challenges.SlugifyTest do
  use ExUnit.Case, async: true

  test "slugify" do
    assert Challenges.Slugify.slugify("My Test Title") == "my-test-title"
    assert Challenges.Slugify.slugify("String w/ some extra &stuff'") == "string-w--some-extra--stuff"
  end

  test "deslugify" do
    assert Challenges.Slugify.deslugify("my-test-title") == "My Test Title"
  end
end
